sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -35,7 +35,6 @@
 !> 2022-07-11 | Jesse Meng | CALSLR_ROEBBER
 !> 2023-02-14 | Jesse Meng | CALSLR_UUTAH     
 !> 2023-03-22 | Sam Trahan | Fix out-of-bounds access by not calling BOUND
-!> 2025-05-05 | Ben Blake  | Add sanity checks for RRFSv1 implementation
```

Comment:
@JesseMeng-NOAA Could you re-add this comment that was removed?  I added it as part of PR #1196 .  Thanks!